test/performance/Benchmarks.md

```diff
@@ -0,0 +1,164 @@
+# Benchmarks
+
+Knative performance benchmarks are tests geared towards producing useful
+performance metrics of the knative system. All the raw metrics are stored in
+[mako](https://github.com/google/mako)
+
+## Writing new benchmarks
```

Comment:
Looks like there's a whole bunch of stuff we can drop from the readme/docs - since they assume results will go to the mako service

Reply:
removed